caikit/runtime/service_generation/protoable.py

```diff
@@ -124,25 +124,33 @@ def is_protoable_type(arg_type: Type) -> bool:
     Or if it's an imported Caikit data model class.
     Or if it's a Union of at least one of those.
     Or if it's a List of one of those.
+    Or if it's a Dict of one of those.
```

Comment:
Should this be tested in `test_protoable.py`? I'd expect some tests of `to_protoable_signature` for valid and invalid dict cases